solution_runner/commands/commands_utils.py

```diff
@@ -0,0 +1,21 @@
+from typing import Any
+
+import click
+import yaml
+
+import consts
+
+
+def get_setting(key: str) -> Any:
+    """
+    :param key: key to retrieve its value
+    :return: value of `key` which is stored in the configuration file
+    """
+    configuration_file = consts.APP_DATA_DIRECTORY / consts.CONFIGURATION_FILE_NAME
+    try:
+        configuration = yaml.safe_load(configuration_file.read_text())
+    except FileNotFoundError:
+        click.secho("configuration file doesn't exist. Run config command first", fg='red')
+        raise
+
+    return configuration[key]
```

Comment:
```suggestion
    return configuration[key]

```
Also, decide if you are using `'` or `"` for strings.

You should use a formatter like yapf or black to do all of that for you.

Reply:
Thanks! I'll set up Black in another PR.